features/environment.py

```diff
@@ -42,5 +42,4 @@ def get_firefox():
     """Creates a headless Firefox driver"""
     options = webdriver.FirefoxOptions()
     options.add_argument("--headless")
-    return webdriver.Firefox(options=options)    
-    
\ No newline at end of file
+    return webdriver.Firefox(options=options)
```

Comment:
What happens if you try adding

options.add_argument("--no-sandbox") in this method?

Reply:
Didn't work

Reply:
:/